web/src/translations/translations.json

```diff
@@ -217,5 +217,114 @@
         "Update": "更新",
         "Delete": "削除"
       }
+    },
+    "da": {
+        "App": {
+          "title": "Get5 Web Panel",
+          "AllMatches": "Alle Kampe",
+          "MyMatches": "Mine Kampe",
+          "MyTeams": "Mine Hold",
+          "CreateMatch": "Opret en kamp",
+          "CreateTeam": "Opret et hold",
+          "MyServers": "Mine Servere",
+          "AddServer": "Tilføj en server",
+          "Logout": "Logud"
+        },
+        "Match": {
+          "AdminTools": "Administrationsværktøjer",
+          "PauseMatch": "Pause kamp",
+          "UnpauseMatch": "Unpause match",
+          "AddPlayerToTeam1": "Føj spiller til team1",
+          "AddPlayerToTeam2": "Føj spiller til team2",
+          "AddPlayerToSpec": "Føj spiller til tilskuerliste",
+          "SendRcon": "Send rcon kommado",
+          "LoadBackupFile": "Load a backup file",
+          "CancelMatch": "Annuller kamp",
+          "MatchHasBeenCancelled": "Denne kamp er annulleret.",
+          "MatchForfeitedBy": "Denne kamp blev fortabt af {team}.",
+          "MatchPendingStart": "Denne kamp afventer start."
```

Comment:
Add translations for "MatchPendingStart", "MessageCancelSuccess", "MessageAddPlayerSuccess", "MessageSendCommandSuccess".
https://github.com/FlowingSPDG/get5-web-go/blob/bb12e7a568a036511eec7be54226a96a12a365b8/web/src/translations/translations.json#L26-L29

Reply:
Fixed!